src/app/dataset-view/additional-components/flows-component/components/tile-base-widget/tile-base-widget.component.html

```diff
@@ -29,7 +29,7 @@
 
                 <span class="fs-12 fw-bold">Activation time:</span>
                 <span class="fs-12 no-wrap">
-                    {{ node.timing.activateAt | date: "M/d/yy, h:mm:ss a" }}
+                    {{ node.timing.runningSince | date: "M/d/yy, h:mm:ss a" }}
```

Comment:
This is not an activation time. This is a moment the task actually started running. Consider renaming the field or hiding it, if `runningSince` is undefined.

Reply:
Fixed